vendor/deps/cli-kit/lib/cli/kit/resolver.rb

```diff
@@ -25,8 +26,20 @@ def call(args)
       private
 
       def command_not_found(name)
-        CLI::UI::Frame.open("Command not found", color: :red, timing: false) do
-          $stderr.puts(CLI::UI.fmt("{{command:#{@tool_name} #{name}}} was not found"))
+        CLI::UI::Frame.open(@ctx.message('kit.resolver.command_not_found'),
```

Comment:
I'm apologize if I've missed this before, but we really shouldn't be updating this file (especially not with messages). All of the files under `vendor` should be treated as external to this repo, as they're only added to this repo so it doesn't have any dependencies - they may be replaced with the current code from `cli-kit` at any time, which would undo these changes in the process.

I believe we'll want to make these changes in `lib/shopify-cli/core/help_resolver.rb`, but since the `command_not_found` method is private, I wonder if we'll be able to alter it in `HelpResolver`.

I haven't tested this yet, but if we can't simply extend `command_not_found`, I see a couple of ways around it:
- We may need to change cli-kit to make that method protected instead, and update the CLI dependencies to pull those in.
- The base method calls `raise CLI::Kit::AbortSilent`, so we could maybe abuse it by rescuing it and printing the help based on it. I don't really like this approach, though.